docs/source/developers/guide/step_by_step/finding_issues.rst

```diff
@@ -69,6 +69,7 @@ in the comments.
    When you find a GitHub issue you would like to work on, please mention
    your interest in the comment section of that issue; that way we will know
    you are working on it.
+   Consider `assigning yourself to the issue <../../bug_reports.html#issue-assignment>`_ when the work starts.
```

Comment:
Can we use `:ref:...` markup here?

```suggestion
   Consider :ref:`assigning yourself to the issue <issue-assignment>`_ when the work starts.
```

Reply:
@kou The way you wrote it didn't work out, it doesn't seem to be valid rst markup? I followed the syntax [here](https://devguide.python.org/documentation/markup/) and made the change.

Reply:
Oh, sorry.
I thought that it works. But I'm not an expert of reST.

FYI:

https://devguide.python.org/documentation/markup/#cross-linking-markup

> Alternatively, you can reference any label (not just section titles) if you provide the link text `` :ref:`link text <reference-label>` ``.

Reply:
Ahhhh, so the `_` was the problem.

Reply:
Ah, I should have removed it!